### PR TITLE
improve doc of Database Connections Pooling

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: src/docs/user/references/DataConnectors.adoc [hunk 1]
```diff
@@ -318,7 +318,10 @@ Default: `HTML`
 
 The scheduler should be configured in non-fork mode to execute tasks in a single JVM rather than starting a dedicated JVM to run the task (In $PROACTIVE_HOME/config/scheduler/settings.ini, set pa.scheduler.task.fork=false).
 
-To be executed, each workflow requires the jdbc connector jar file of the used rdbms to connect to the database. Make sure you have it properly copied into the addons directory ($PROACTIVE_HOME/addons) of the scheduler.
+To be executed, each workflow requires the jdbc connector jar of the used rdbms to enable connection to the database.
```
Comment (mklkun) [inline src/docs/user/references/DataConnectors.adoc]: to enable connection to the database. => to enable the connection to the database.

File: src/docs/user/references/DataConnectors.adoc [hunk 2]
```diff
@@ -318,7 +318,10 @@ Default: `HTML`
...
+In case the Scheduler is running with only local Nodes, make sure that you have properly added the jdbc jar into the addons directory ($PROACTIVE_HOME/addons) of the Scheduler before starting the server.
+Otherwise, adding the jdbc connector jar depends on how you will deploy the ProActive Node. If you use the enterprise Node release, make sure you have it properly added into the addons directory ($PROACTIVE_NODE/addons) of the node.
+If you use the node.jar, go under $PROACTIVE_HOME/dist/war of the Scheduler and inject the jdbc jar into the node.jar.
```
Comment (fviale) [inline src/docs/user/references/DataConnectors.adoc]: If you use cloud infrastructures (or any other which use the node.jar mechanism), go in $PROACTIVE_HOME/dist/war/rest on the scheduler server, and add the jdbc jar inside the lib folder of the node.jar archive.

**Comment by fviale** [comment]
1 comment

File: src/docs/user/references/DataConnectors.adoc [hunk 3]
```diff
@@ -318,7 +318,10 @@ Default: `HTML`
 
 The scheduler should be configured in non-fork mode to execute tasks in a single JVM rather than starting a dedicated JVM to run the task (In $PROACTIVE_HOME/config/scheduler/settings.ini, set pa.scheduler.task.fork=false).
 
-To be executed, each workflow requires the jdbc connector jar file of the used rdbms to connect to the database. Make sure you have it properly copied into the addons directory ($PROACTIVE_HOME/addons) of the scheduler.
+To be executed, each workflow requires the jdbc connector jar of the used rdbms to enable connection to the database.
+In case the Scheduler is running with only local Nodes, make sure that you have properly added the jdbc jar into the addons directory ($PROACTIVE_HOME/addons) of the Scheduler before starting the server.
```
Comment (mklkun) [inline src/docs/user/references/DataConnectors.adoc]: is running with only local Nodes => is running on local Nodes only
make sure that you have properly added the jdbc jar into the addons directory ($PROACTIVE_HOME/addons) of the Scheduler before starting the server. => make sure that you have properly added the jdbc jar, before starting the server, into the addons directory ($PROACTIVE_HOME/addons) of the Scheduler.

**Comment by fviale** [comment]
Retest this please

File: src/docs/user/references/DataConnectors.adoc (hunk 2 above)
Comment (mklkun) [inline src/docs/user/references/DataConnectors.adoc]: go under => go to (maybe?)

File: src/docs/user/references/DataConnectors.adoc [hunk 4]
```diff
@@ -318,7 +318,11 @@ Default: `HTML`
 
 The scheduler should be configured in non-fork mode to execute tasks in a single JVM rather than starting a dedicated JVM to run the task (In $PROACTIVE_HOME/config/scheduler/settings.ini, set pa.scheduler.task.fork=false).
 
-To be executed, each workflow requires the jdbc connector jar file of the used rdbms to connect to the database. Make sure you have it properly copied into the addons directory ($PROACTIVE_HOME/addons) of the scheduler.
+Data connectors workflows may need to add some third-party dependencies to their classpath (notably, JDBC jar libraries), so they can execute properly. The user determines the required dependencies depending on the workflow implementation and the used DataBase system. Third-party dependencies can be added to the workflow classpath in three ways (depending on the nodes that will run the workflow):
```
Comment (mklkun) [inline src/docs/user/references/DataConnectors.adoc]: so they can execute properly => so they can be executed properly

**Pull request merged**